### ISSUE TITLE
The screen reader does not read the tooltip contents

### ISSUE THREAD

**Issue by cont-amibabkin** [Esri/calcite-components#13270]
**monday.com sync:** #10513343193

### Check existing issues

- [x] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates and reviewed the [Accessibility page](https://developers.arcgis.com/calcite-design-system/foundations/accessibility) for guidance.

### Which Component

Tooltip

### Summary

The screen reader doesn't read tooltip content in version 3.3.3 of the calcite-components library. Everything works fine in version 3.0.3 of the calcite-components library.

### Actual Behavior

NVDA and VoiceOver don't speak the tooltip in version 3.3.3.

<img width="428" height="304" alt="Image" src="https://github.com/user-attachments/assets/916e4223-d5cd-41ae-ab8d-56697fef7089" />

### Expected Behavior

NVDA and VoiceOver speak the Tooltip contents when focused (opened), but not when focused is removed.

<img width="352" height="304" alt="Image" src="https://github.com/user-attachments/assets/063472f9-43ca-4cef-80db-434bf07e7395" />

### Reproduction Sample

3.3.3: https://codepen.io/Andrey-B-the-animator/pen/emJaVpO?editors=100 
3.0.3: https://codepen.io/Andrey-B-the-animator/pen/YPwbeyQ?editors=100

### Reproduction Steps

1. Activate screen reader.
2. Ensure that the screen reader doesn't read the tooltip.

### Reproduction Version

3.3.3

### Priority impact

impact - p2 - want for an upcoming milestone

### Calcite package

- [x] @esri/calcite-components
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

ArcGIS Business/Community Analyst

**Comment by isaacbraun** [comment]
Spike to check if this is still a issue and confirm regression. If a fix is needed, it would be p-high and est-5.

**Comment by isaacbraun** [comment]
Spike completed with multiple findings:
1. On `3.0.3`:
	1. NVDA does read out the tooltip contents when opened and/or closed.
	2. VoiceOver did not read out the tooltip contents, unless specifically focused. @cont-amibabkin Were you focusing the tooltip with `Ctrl + Opt + Shift + Right Arrow` or was it reading out just on focus of the `calcite-link`? 
2. On `3.3.3` and `5.0.0-next.86`, NVDA does **not**  read out the tooltip contents.

This does look like a regression and should be fixed.

For implementation, @geospatialem should the tooltip contents be read out when it is opened **and** closed, just when opened, or something else?

**Comment by HauTruong146** [comment]
Hi @isaacbraun, navigating through the Calcite tooltip page with the VoiceOver on Mac, the tooltip announces correctly when the Calcite action is in focus, but not for the Calcite link. To get the screen reader to announce the tooltip, as you mentioned, we will need to set the focus on the tooltip using the special keyboard navigation. 

https://github.com/user-attachments/assets/548691bc-df0e-4ac5-80ae-707be9c96120

After discussing with @cont-amibabkin, we would love to have the tool tip announced for the link as well, without having to specifically focused on.

**Comment by geospatialem** [comment]
> For implementation, [@geospatialem](https://github.com/geospatialem) should the tooltip contents be read out when it is opened **and** closed, just when opened, or something else?

@isaacbraun Great question, we only need to provide the context upon opening per [Success Criterion1.4.13: Content on Hover or Focus (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html). The intent of the criterion is so users can perceive (via the component's contents) and dismiss non-persistent content.

**Comment by yuhyun-kwon** [comment]
Adding a case to this issue. Testing a11y on newly implemented feature, Items used, and faced inconsistent behavior on tooltips. (Reported by our a11y person Devi)
-  Mac voiceover: Reads the tooltip as it is focused (expected)
- NVDA: Did not read the tooltip as it is focused, but reads it after tabbed out from the tooltip (This might be issue on Windows)
- JAWS: Reads the tooltip but not the button/chip name

<img width="1076" height="265" alt="Image" src="https://github.com/user-attachments/assets/bc4b9aad-b22a-4f8c-9e26-25e7695d2d23" />

To reproduce
1. Create a web map and include an item (e.g. feature layer) with status badges (e.g. deprecated)
2. Save the map and navigate to its item details page
3. Test screen readers by fodcusing on the status badge in the list item

**Comment by HauTruong146** [comment]
Adding a new case to this issue. While using NVDA, when link/action are in focus, the tooltip is announced twice. 

https://github.com/user-attachments/assets/008f4d74-39e2-40ad-9029-743d902a476b

**Comment by ddred11** [comment]
@isaacbraun Hello! Wondering if this can be moved up in priority? This issue is one of our dependencies for the R3 BA release.
